dev-aws/kafka-shared-msk/data-infra/eqdb-loader.tf

```diff
@@ -1,7 +1,7 @@
 resource "kafka_topic" "di_eqdbloader_schedule" {
   name               = "data-infra.eqdb-loader.schedule"
   replication_factor = 3
-  partitions         = 1
+  partitions         = 12
```

Comment:
Why this topic needs so many partitions?

Reply:
Because we will use it to distribute the load for eqdb-loader
we can have up to 12 writers reading from it directly.